app/controllers/system_admin/lead_partners_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  class LeadPartnersController < ApplicationController
+    before_action { require_feature_flag(:lead_partners) }
+
+    def index
+      @lead_partners = LeadPartner.order(:name).page(params[:page] || 1)
```

Comment:
This is missing the `Policy` scope similar to [SystemAdmin::LeadSchoolsController#index](https://github.com/DFE-Digital/register-trainee-teachers/blob/7168-add-lead-partner-and-related-on-the-support-console/app/controllers/system_admin/lead_schools_controller.rb#L6)